indico/modules/events/surveys/util.py

```diff
@@ -139,4 +139,8 @@ def query_active_surveys(event):
     if session.user:
         user_pending_criterion = ~SurveySubmission.is_submitted & (SurveySubmission.user == session.user)
         private_criterion |= Survey.submissions.any(user_pending_criterion)
-    return Survey.query.with_parent(event).filter(Survey.is_active, private_criterion)
+    return (
```

Comment:
Shouldn't the wrapping be like this in python?
```python
return (Survey.query.with_parent(event)
        .filter(Survey.is_active, private_criterion)
        .order_by(db.func.lower(Survey.title)))
```

Reply:
I guess it was `black` formatting it this way..

Reply:
I wasn't really sure about the wrapping but this was suggested by black. I decided to follow black since I wasn't sure.

Reply:
Oh, didn't know we were using `black` in Indico

Reply:
We are not.. I just wanted a suggestion on a formatting and I used that tool for it. If you want I can change it to the one you suggested.

Reply:
Actually @mvidalgarcia is right. We are not using `black` for Indico for now.

Reply:
Changed it to @mvidalgarcia 's suggestion. As I explained, I wasn't trying to add `black` in indico and I know we are not using it. I just took this suggestion as I was unsure about indentation at that part.

Reply:
black in indico is not gonna happen for now (but something to consider when we go for python 3) - so best to not have it enabled in your editor either.

Anyway, I think asking tools like black and prettier for advice when you are unsure on how to indent something is generally not a bad idea. However for chained calls in queries we already have a good style we use everywhere in Indico ;)